dev/com.ibm.ws.threading/test/com/ibm/ws/threading/internal/PolicyExecutorTest.java

```diff
@@ -76,24 +76,34 @@ public Integer call() throws Exception {
     public void testExpedite() throws Exception {
         ExecutorServiceImpl globalExecutor = new ExecutorServiceImpl();
         Map<String, Object> globalExecutorConfig = new HashMap<String, Object>();
-        globalExecutorConfig.put("coreThreads", 1);
-        globalExecutorConfig.put("maxThreads", 1);
-        globalExecutorConfig.put("keepAlive", TimeUnit.MINUTES.toMillis(5));
+        globalExecutorConfig.put("coreThreads", ExecutorServiceImpl.MINIMUM_POOL_SIZE);
+        globalExecutorConfig.put("maxThreads", ExecutorServiceImpl.MINIMUM_POOL_SIZE);
+        // Jan-2019: remove keepAlive, no longer supported by the default executor
+        // globalExecutorConfig.put("keepAlive", TimeUnit.MINUTES.toMillis(5));
         globalExecutor.activate(globalExecutorConfig);
-        globalExecutor.threadPoolController.deactivate(); // disable autonomic tuning to guarantee maxThreads stays the same
+        // Jan 2019: deactivate no longer required, threadPoolController automatically deactivated if (coreThreads == maxThreads)
+        //globalExecutor.threadPoolController.deactivate(); // disable autonomic tuning to guarantee maxThreads stays the same
```

Comment:
Again, remove commented-out code and historical part of comment

Reply:
Agreed, done.